src/index.html

```diff
@@ -33,7 +33,7 @@
                 <div class="sidetab-top">
                     <img src="assets/svg/search_tab.svg" class="top-icon" />
                     <input class="sidetab" type="text" maxlength="20" placeholder="Enter stock name or symbol"
-                           ng-show="sidebarCtrl.showSearches()" ng-model="searchCtrl.query" />
+                           ng-show="sidebarCtrl.showSearches()" ng-model="searchCtrl.query" ng-keyup="searchCtrl.onSearchKeyUp($event)" />
                 </div>
```

Comment:
That's really quite frustrating. I'd like to see functionality where a user can press the down key and the selection will initially move one down, and then start to go down the list. Maybe some sort of setInterval? But surely this is a solved problem...

In any case keyDown feels a more natural experience for me..